packages/core/src/RenderingEngine/helpers/cpuFallback/rendering/getVOILut.ts

```diff
@@ -15,6 +15,10 @@
  */
 
 /**
+ * Generates the linear VOI LUT function.
+ * From the DICOM standard, that is:
+ * ((x - (c - 0.5)) / (w-1) + 0.5) * (ymax- ymin) + ymin
+ * clipped to the ymin...ymax range
```

Comment:
can we have some links to the standard on why this is the formula? and how you are sure this is correct? since this is significant change that we didn't even have in v2 or cornerstone legacy

Reply:
It is the definition in the standard - the other one can be made to work, but they tweaked the definition in the standard in order to have 2 things:
1. Make the width be the count of values rather than the absolute length
2. Make the center point of an even width (as in #1) be an integer value

That makes it easy to represent the 2^n bit lengths with an identity window level/width, as the number of points is 2^n and the center being 2^(n-1) as opposed to 2^n-1 and 2^(n-1)-0.5

Added the link.

Reply:
Note that it is the same function, just that width and center are offset by 1 and 0.5 respectively.